src/operations.rs

```diff
@@ -55,8 +56,7 @@ pub(crate) fn shuffle(a: u128) -> u128 {
         use core::arch::x86::*;
         #[cfg(target_arch = "x86_64")]
         use core::arch::x86_64::*;
-        use core::mem::transmute;
-        unsafe { transmute(_mm_shuffle_epi8(transmute(a), transmute(SHUFFLE_MASK))) }
+        unsafe { transmute!(_mm_shuffle_epi8(transmute!(a), transmute!(SHUFFLE_MASK))) }
```

Comment:
This didn't reduce the scope of `unsafe` like #171.

IDK if you run into performance issues doing so, but I would investigate doing something like:
```rust
#[cfg(target_arch = "x86_64")]
type Vector = __m128;
#[inline(always)]
const fn scalar(x: Vector) -> u128 {
    unsafe { transmute(x) }
}
#[inline(always)]
const fn vector(x: u128) -> Vector {
    unsafe { transmute(x) }
}
```

```rust
const SHUFFLE_MASK: Vector = vector(SHUFFLE_MASK);
let a = vector(a);
scalar(unsafe { _mm_shuffle_epi8(a, SHUFFLE_MASK) })
```
IMO, this meets the goal of clarifying that the `unsafe` conversions are safe with minimal work, without a dependency.